src/viewer.js

```diff
@@ -561,6 +577,53 @@ $.extend( $.Viewer.prototype, $.EventSource.prototype, $.ControlDock.prototype,
         return !!this.world.getItemCount();
     },
 
+    /**
+     * Checks whether all TiledImage instances in the viewer's world are fully loaded.
+     * This determines if the entire viewer content is ready for optimal display without partial tile loading.
+     * @returns {Boolean} True if all TiledImages report being fully loaded, false if any image still has pending tiles
+     */
+    areAllFullyLoaded: function() {
```

Comment:
I think this should be private, since we want users of the API to just call `getFullyLoaded`. Include a `@private` tag in the doc comment and also add an underscore to the beginning of the function name.